orca-spring-batch/src/main/java/com/netflix/spinnaker/config/SpringBatchConfiguration.java

```diff
@@ -77,10 +78,10 @@ ExecutionListenerProvider springBatchStepExecutionListenerProvider(ExecutionRepo
 
   @Bean
   StageBuilderProvider springBatchStageBuilderProvider(ApplicationContext applicationContext,
-                                                       Collection<StageBuilder> stageBuilders,
+                                                       Optional<Collection<StageBuilder>> stageBuilders,
```

Comment:
There are no StageBuilders left in orca-core now so tests started failing because they couldn't autowire this